client/src/partials/inventory/configuration/units/units.js

```diff
@@ -25,35 +25,34 @@ function InventoryUnitsController(InventoryUnit, Notify, Modal) {
     var request = { action : 'add' };
 
     Modal.openInventoryUnitActions(request)
-    .then(function (res) {
-      if (res.id) {
-        Notify.success('FORM.INFO.CREATE_SUCCESS');
-      }
-    })
-    .then(startup)
-    .catch(Notify.errorHandler);
+      .then(function (res) {
+        if (res.id) {
```

Comment:
Do you know what ` if (res.id)` does for you here? What if there is no `id` on `res`?